assignment-4/index.ts

```diff
@@ -0,0 +1,52 @@
+// Requiring Modules
+import express from 'express';
```

Comment:
I've got a few errors while compiling project with tsc

routes/router.ts:295:52 - error TS2339: Property 'params' does not exist on type 'IRequest'.

295                       res.redirect('/notes/' + req.params.id);
                                                       ~~~~~~


Found 47 errors in 3 files.

Errors  Files
     1  index.ts:2
     9  routes.test.ts:4
    37  routes/router.ts:1